scripts/core/registry.yml

```diff
@@ -592,6 +592,12 @@ etors:
 - name: ENQUEUE_NATIVE_COMMAND_EXP
   desc: Enumerator for $xEnqueueNativeCommandExp
   value: '228'
+- name: ADAPTER_SET_LOGGER_CALLBACK
+  desc: Enumerator for $xAdapterSetLoggerCallback
+  value: '231'
+- name: ADAPTER_SET_LOGGER_CALLBACK_LEVEL
+  desc: Enumerator for $xAdapterSetLoggerCallbackLevel
+  value: '232'
```

Comment:
This need reset, changing the entry point names will have done this.

Reply:
Oh strange why would it increment the value rather than just updating the name?

Reply:
The script doesn't know the difference between a new entry point having a name change and an existing entry point being removed. The ID's are intended to be globally unqiue for use in external tooling.